db/seeds/sandbox_data.rb

```diff
@@ -38,14 +38,14 @@ def create_school_and_associations(lead_provider, cohort, index)
     s.postcode = Faker::Address.postcode
   end
 
-  Partnership.create!(
+  Partnership.find_or_create_by!(
```

Comment:
This is never going to match, because the delivery partner is randomly generated, right?